recursive_count_th/count_th.py

```diff
@@ -3,8 +3,37 @@
 Your function should return a count of how many occurences of ***"th"*** occur within `word`. Case matters.
 Your function must utilize recursion. It cannot contain any loops.
 '''
+
+#set 'th' counter
+
+th_count = 0
+
+
 def count_th(word):
-    
-    # TBC
-    
-    pass
+    #reset count after each run
+    global th_count
+    th_count = 0
+    #get length of word
+    word_l = len(word)
```

Comment:
this can be done with a single recursive function, try to refactor when you get a chance!